go-controller/pkg/cluster/node_test.go

```diff
@@ -292,83 +271,5 @@ var _ = Describe("Node Operations", func() {
 		Expect(err).NotTo(HaveOccurred())
 
 	})
-	It("test watchConfigEndpoints multiple IPs", func() {
```

Comment:
@alexanderConstantinescu aren't we going to want this test back when we get RAFT? Or do we just resurrect it from git history...

Reply:
Indeed, I did this while being swept up in Numan's PR and thus forgot to stop, breath and realize this fact. This has already been reverted on my branch for the on-going work on raft.